runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -411,4 +411,7 @@
 
 	@ScriptArguments(integer = 9)
 	public static final int WORLDMAP_LOADMAP = 1712;
+
+	@ScriptArguments(integer = 7, string = 1)
+	public static final int CHARACTER_SUMMARY_TEXT_FORMAT = 3948;
```

Comment:
Let's call this "account summary" text summary to match <https://github.com/Joshua-F/cs2-scripts/blob/afbd80d369facdcbbe01271dbc064f97924f6d0a/scripts/%5Bclientscript%2Caccount_summary_update_combatlevel%5D.cs2> and <https://github.com/Joshua-F/cs2-scripts/blob/afbd80d369facdcbbe01271dbc064f97924f6d0a/scripts/%5Bclientscript%2Caccount_summary_update_totallevel%5D.cs2>
```suggestion
	public static final int ACCOUNT_SUMMARY_TEXT_FORMAT = 3948;
```

Reply:
Updated, thanks! Would you like me to update the widget and variable names throughout this pull request from "character" to "account" as well?

Note that the reason I originally chose "character" rather than "account" is that when you mouse over the button to change to that tab, the mouseover text is "Character Summary".